src/math/p5.Vector.js

```diff
@@ -1549,7 +1549,10 @@ p5.Vector.prototype.rotate = function rotate(a) {
 };
 
 /**
- * Calculates and returns the angle (in radians) between two vectors.
+ * Calculates and returns the angle between two vectors. This function will take
+ * the current <a = src="#/p5/angleMode">angleMode</a> into consideration, and
```

Comment:
thanks @sflanker! this looks good, but we need to use `href` for the link to work
```suggestion
 * the current <a href="#/p5/angleMode">angleMode</a> into consideration, and
```

Reply:
Oops, I copied this from the comments on the `heading` function without paying enough attention.

Reply:
no problem! thank you!